veggieseasons/lib/widgets/search_bar.dart

```diff
@@ -39,6 +39,12 @@ class SearchBar extends StatelessWidget {
               child: CupertinoTextField(
                 controller: controller,
                 focusNode: focusNode,
+                 decoration: BoxDecoration( // to remove white area behind text field
```

Comment:
@LongCatIsLooong/@xster, what's your thought here?

Is this a flaw in how the original code is using CupertinoTextField?